### PR TITLE
init

### PR DESCRIPTION
**Pull request by danieTalmon**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+                <div class="description">
+                  Digital javascript book to start coding
+                  your first js website
+                </div>
+              </div>
+              <div class="extra content">
+                <div class="ui list">
+                  <div class="item">  
+                    <span class="left floated">
+                      ₪330.00
```
Comment (edenmarom) [inline index.html]: currency should be implemented using css

File: index.html [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+        </div>
+        <div class="right floated center aligned four wide column padding-bottom-zero">
+          <div class="ui inverted segment" style=" padding-bottom: 0;">
+              <div class="ui inverted compact text  menu padding-bottom-zero">
+              <a id="browse-item" class="browse item bottom">
+                  Account&Lists
+                  <i class="dropdown white icon compact-icon"></i>
+              </a>
+              <a id="cart" class=" item">
+                <span><h5 id="prodect-count" class="ui orange header inverted aligned top">(2)</h5>
```
Comment (edenmarom) [inline index.html]: "(", ")" should be implemented using css

File: index.html [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+      <div class="column">
+        <h4 class="ui header">Your List</h4>
+        <div class="ui link list">
+          <a class="item">Create New List</a>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="ui large modal">
+    <div class="header">
```
Comment (edenmarom) [inline index.html:74]: You should implement a dropdown in he cart.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+                <p>Good coconut make oil for your beard </p>
+              </div>
+              <div class="item">
+                <p>
+                  ₪100.00
+                </p>
+              </div>
+              <div class="item">
+                <div class="ui selection dropdown">
+                  <input name="gender" type="hidden">
```
Comment (edenmarom) [inline index.html:103]: there is a difference between the 2 types of inputs

File: styles/navbar.less [hunk 5]
```diff
@@ -0,0 +1,53 @@
+@margin-zero: 0;
```
Comment (edenmarom) [inline styles/navbar.less]: no need for his variable

File: styles/navbar.less [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+@padding-zero:0;
```
Comment (edenmarom) [inline styles/navbar.less]: no need for his variable

File: semantic/components/accordion.css [hunk 7]
```diff
@@ -0,0 +1,253 @@
+/*!
+ * # Semantic UI 2.4.1 - Accordion
+ * http://github.com/semantic-org/semantic-ui/
+ *
+ *
```
Comment (edenmarom) [inline semantic/components/accordion.css]: don include the semantic input just he themes file.

File: index.html (hunk 2 above)
Comment (edenmarom) [reply index.html]: ?

**Comment by edenmarom** [comment]
General notes:
3. input of secound product in cart is not working.
4. use semantic's themes page. 
5. some css elements are in the nav bar.less file even when the are related to the card.less file.
6. acounts and list styiling should be fixed. also open on hover.
7.there are 4 card in a row.
8. nav bar is thinner.
9. styling on freedom card should be fixed.
10. each card length is set by the longest card.
11. cart modal size should be smaller.
Good Luck!

File: styles/navbar.less [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+@link-hover-color: rgb(172, 162, 32);
+
+
+#product-count {
+    margin-bottom: @margin-zero;
+    padding-right: 0.4rem;
+    margin-right: 7.5rem;
```
Comment (alon6699) [inline styles/navbar.less]: do not use rem everywhere ... uuse %

File: index.html [hunk 9]
```diff
@@ -0,0 +1,363 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>amaRomach</title>
+    <link rel="shortcut icon" href="images/amazon-icon.jpg" type="image/x-icon">
+    <link rel="stylesheet" type="text/css" href="semantic/dist/semantic.min.css">
+    <link rel="stylesheet" type="text/css" href="styles/navbar.css">
```
Comment (alon6699) [inline index.html]: import all css to one file us @import less

File: index.html [hunk 10]
```diff
@@ -0,0 +1,363 @@
...
+      src="https://code.jquery.com/jquery-3.1.1.min.js"
+      integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+      crossorigin="anonymous"></script>
+    <script src="semantic/dist/semantic.min.js"></script>
+    <script src="js/amaRomach.js"></script>
+</head>
+<body>
+  <div class="ui inverted segment">
+    <div class="ui grid">
+        <div class="row padding-bottom-zero">
```
Comment (alon6699) [inline index.html]: rename to no-padding-bottom

File: index.html [hunk 11]
```diff
@@ -0,0 +1,363 @@
...
+                    <button class="ui primary button" >
+                        <i class="ui large search icon"></i>
+                    </button>
+                </div>
+                </div>
+        </div>
+        </div>
+        <div class="row padding-bottom-zero">
+          <div class="twelve wide column padding-right-zero"></div>
+          <div id="browse-container" class="ui right aligned two wide column">
```
Comment (alon6699) [inline index.html]: try to use float in grid

File: index.html [hunk 12]
```diff
@@ -0,0 +1,363 @@
...
+            <div class="ui inverted segment padding-bottom-zero padding-right-zero padding-left-zero">
```
Comment (alon6699) [inline index.html]: get background from parent

File: styles/cart.css [hunk 13]
```diff
@@ -0,0 +1,39 @@
+#product-count {
+  margin-bottom: 0;
+  margin-right: 79%;
+  vertical-align: top;
+}
+#cart {
```
Comment (alon6699) [inline styles/cart.css:6]: remove

File: index.html [hunk 14]
```diff
@@ -0,0 +1,363 @@
...
+                    </div>
+                  </div>
+                </div>
+                </div>
+              </div>
+            </div>
+              </div>
+              <div id="cart-grid" class="ui right aligned two wide column">
+                  <span id="product-count" class="parenthesis ui orange header">2</span>
+                  <div id="cart-btn" class="inline cart-link">
```
Comment (alon6699) [inline index.html]: rename cart - move all cart-link style to #cart

File: styles/navbar.less [hunk 15]
```diff
@@ -0,0 +1,46 @@
+@margin-zero: 0;
+@padding-zero:0;
+@link-hover-color: rgb(172, 162, 32);
```
Comment (alon6699) [inline styles/navbar.less]: use hex and not rgb like #FFAA00

File: styles/cart.less [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+  }
+
+.cart-header {
+    width: 50%;
+    padding-right: 13%;
+    float: right;
+  }
+
+.parenthesis {
+    color: #ff851b !important;
```
Comment (alon6699) [inline styles/cart.less]: remove it from here

File: index.html [hunk 17]
```diff
@@ -0,0 +1,363 @@
...
+      </div>
+      <div class="item amaRomach-cart-item">
+        <div class="image">
+          <img src="images/jsBook.jfif">
+        </div>
+        <div class="content">
+          <div class="ui huge header"> JS book for beginners</div>
+          <div class="ui right floated"><i class="ui small red close icon red-close-icon"></i></div>
+          <div class="description">
+            <div class="ui list">
```
Comment (alon6699) [inline index.html]: try to use grid

File: index.html [hunk 18]
```diff
@@ -0,0 +1,363 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="item amaRomach-cart-item">
+        <div class="image">
+          <img src="images/jsBook.jfif">
+        </div>
+        <div class="content">
+          <div class="ui huge header"> JS book for beginners</div>
+          <div class="ui right floated"><i class="ui small red close icon red-close-icon"></i></div>
```
Comment (alon6699) [inline index.html]: fix this

File: styles/cards.less [hunk 19]
```diff
@@ -0,0 +1,22 @@
...
+    flex-basis: 0 !important;
+}
+
+#total-price {
+    margin-left: 6% !important;
+}
+
+.amaRomach-cart-item {
+    margin-left: 1% !important;
+    font-size: 50% !important;
```
Comment (alon6699) [inline styles/cards.less]: em

File: styles/cards.less [hunk 20]
```diff
@@ -0,0 +1,22 @@
+.amaRomach-card {
+    flex: 1 !important;
+    flex-basis: 0 !important;
```
Comment (alon6699) [inline styles/cards.less:3]: explain

File: styles/navbar.css [hunk 21]
```diff
@@ -0,0 +1,34 @@
+body {
+  min-width: 1075px;
```
Comment (alon6699) [inline styles/navbar.css:2]: remove or change

File: styles/cards.less (hunk 20 above)
Comment (danieTalmon) [reply styles/cards.less:3]: It means the basis of the flexbox so zero will collapse the element to its smallest possible width given the content

File: styles/navbar.css (hunk 21 above)
Comment (danieTalmon) [reply styles/navbar.css:2]: Eden told me that its nesscery..

File: styles/cart.less [hunk 22]
```diff
@@ -0,0 +1,48 @@
...
+
+    &::before {
+        content: "\0028" !important ;
+    }
+    &::after {
+        content: "\0029" !important ;
```
Comment (alon6699) [inline styles/cart.less]: change to (

File: styles/cards.less [hunk 23]
```diff
@@ -0,0 +1,21 @@
...
+.amaRomach-cart-item {
+    padding-left: 0.5% !important;
+}
+
+.header-link:hover {
+    color: #aca220 !important;
+    text-decoration: underline !important;
+}
+
+.card-image:hover {
```
Comment (alon6699) [inline styles/cards.less]: use &

File: styles/cards.less [hunk 24]
```diff
@@ -0,0 +1,21 @@
...
+
+#total-price {
+    margin-left: 6% !important;
+}
+
+.amaRomach-cart-item {
+    padding-left: 0.5% !important;
+}
+
+.header-link:hover {
```
Comment (alon6699) [inline styles/cards.less]: use &

File: styles/cart.less [hunk 25]
```diff
@@ -0,0 +1,34 @@
+#product-count {
+  margin-bottom: 0;
+  margin-right: 79%;
```
Comment (alon6699) [inline styles/cart.less:3]: change to 80 - round it all

File: styles/cart.less [hunk 26]
```diff
@@ -0,0 +1,34 @@
...
+  vertical-align: top;
+}
+
+#cart {
+  width: 75%;
+  padding-right: 29%;
```
Comment (alon6699) [inline styles/cart.less:9]: 30

File: styles/cards.less [hunk 27]
```diff
@@ -0,0 +1,21 @@
...
+    flex: 1 !important;
+    flex-basis: 0 !important;
+}
+
+#total-price {
+    margin-left: 6% !important;
+}
+
+.amaRomach-cart-item {
+    padding-left: 0.5% !important;
```
Comment (alon6699) [inline styles/cards.less]: try to remove as much important as you can

File: index.html [hunk 28]
```diff
@@ -0,0 +1,362 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="item amaRomach-cart-item">
+        <div class="image">
+          <img src="images/jsBook.jfif">
+        </div>
+        <div class="content">
+          <div class="ui large header"> JS book for beginners</div>
+          <div class="ui right floated"><i class="ui large red close icon red-close-icon"></i></div>
```
Comment (alon6699) [inline index.html]: red-close-icon is your class? if so remove it because it is not defined

File: styles/cards.less (hunk 27 above)
Comment (alon6699) [inline styles/cards.less]: why do you need so little padding?  remove it

File: styles/navbar.less [hunk 29]
```diff
@@ -0,0 +1,44 @@
...
+.no-padding-bottom {
+    padding-bottom: 0 !important;
+}
+
+.no-padding-left {
+    padding-left: 0 !important;
+}
+
+.compact-icon {
+    margin-left: 0.5% !important;
```
Comment (alon6699) [inline styles/navbar.less:38]: why do you need so little margin?

File: index.html [hunk 30]
```diff
@@ -0,0 +1,362 @@
...
+      crossorigin="anonymous"></script>
+    <script src="semantic/dist/semantic.min.js"></script>
+    <script src="js/amaRomach.js"></script>
+</head>
+<body>
+  <div class="ui inverted segment">
+    <div class="ui grid">
+        <div class="row no-padding-bottom">
+            <div class="four wide column">
+                <div id="amaRomach-logo" class="ui inverted segment">
```
Comment (alon6699) [inline index.html]: remove inverted segment

**Pull request merged**